setup.py

```diff
@@ -37,6 +37,7 @@
     ],
     entry_points={
             'console_scripts': [
+                'AttachBarcodes = sctools.platform:BarcodePlatform.attach_barcodes',
```

Comment:
Can this be called something more descriptive? `AttachBarcodes` is so general that 10x users might think to use it for their method.

Reply:
This is more general than the 10x, so users could use this as well. Is the a place where I could provide more explicit documentation explaining the difference between the two classes? (10xV2 has hardcoded values for the barcode start positions/lengths for that particular assay rather than requiring users to provide barcodes start positions/lengths for the ones they want)